src/api/public/apidocs/paths/status_reports_built_project_name_repository_name_architecture_name_reports_build_id.yaml

```
@@ -0,0 +1,75 @@
+get:
+  summary: Get list of checks in the built project repository
+  description: Get a full list of checks in the built project repository's status reports
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/repository_name.yaml'
+    - $ref: '../components/parameters/architecture_name.yaml'
+    - $ref: '../components/parameters/build_id.yaml'
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/status_report.yaml'
+          example:
+            check:
+              - name: First
+                required: false
+                state: pending
+                short_description: This is the first of the checks
+                url: https://ci.opensuse.org/
+              - name: Second
+                required: true
+                state: success
+                short_description: This is the second of the checks
+                url: https://check.opensuse.org/
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      $ref: '../components/responses/not_found.yaml'
+  tags:
+    - Status Reports
+
+post:
+  summary: Submit a check to a built project repository's status report
+  description: Submits a new or modified check to the status report for the built project repository
```

```suggestion
  description: Create or update a check to the status report for the built project repository
```
"Submits a new or modified check..." doesn't sound to me accurate enough. What do you think?

Same for the two other cases (`published/` and `requests/`).